src/python/dxpy/toolkit_version.py

```diff
@@ -1 +1 @@
-version = '0.370.0'
+version = '0.370.0-13-gb5b7342e6'
```

Comment:
can you please revert version? I think we no longer use automatic bump...am I right @kpjensen ?